package/MDAnalysis/coordinates/TRR.py

```diff
@@ -105,7 +105,7 @@ def _write_next_frame(self, ag):
                 self.convert_forces_to_native(forces)
 
         time = ts.time
-        step = ts.frame
+        step = ts.data.get('step', ts.frame)
```

Comment:
Can you add a versionchanged entry please?

Can you maybe update the blurb on line 48? Whilst you're here, can you fix the typo on line 44 ("velocoties").

Reply:
Fixed two typos ('velocoties'), added versionchanged for 2.1.0, and appended to the docstring following line 48 as an example.